hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieWriteStat.java

```diff
@@ -213,6 +242,46 @@ public String getTempPath() {
     return this.tempPath;
   }
 
+  public Long getTotalLogFilesSize() {
+    return totalLogFilesSize;
+  }
+
+  public void setTotalLogFilesSize(Long totalLogFilesSize) {
+    this.totalLogFilesSize = totalLogFilesSize;
+  }
+
+  public Long getTotalLogBlocks() {
+    return totalLogBlocks;
+  }
+
+  public void setTotalLogBlocks(Long totalLogBlocks) {
+    this.totalLogBlocks = totalLogBlocks;
+  }
+
+  public Long getTotalCorruptLogBlock() {
+    return totalCorruptLogBlock;
+  }
+
+  public void setTotalCorruptLogBlock(Long totalCorruptLogBlock) {
+    this.totalCorruptLogBlock = totalCorruptLogBlock;
+  }
+
+  public Long getTotalRollbackBlocks() {
+    return totalRollbackBlocks;
+  }
+
+  public void setTotalRollbackBlocks(Long totalRollbackBlocks) {
+    this.totalRollbackBlocks = totalRollbackBlocks;
+  }
+
+  public long getTotalTimeTakenByLogScanner() {
```

Comment:
I dont know if performance related measures should be part of commit metadata..

Reply:
Hmm, do we want to emit them as graphana metrics ?

Reply:
yeah. we should keep commit metadata purely about the actual commit.. Can you please pull out the perf related metrics from here?